src/button-group/menu-dropdown-item.tsx

```diff
@@ -60,6 +57,9 @@ const MenuDropdownItem = React.forwardRef(
               ariaExpanded={ariaExpanded}
               className={clsx(testUtilStyles.item, testUtilsClass)}
               iconName="ellipsis"
+              loading={item.loading}
+              loadingText={item.loadingText}
+              disabled={item.disabled}
```

Comment:
I created a new unit test for this change but it is also covered with item-permutations. Currently the menu item is rendered the same even when it is set as loading or disabled.